### PR TITLE
Added Idinio Dimmer-Switch-ZB3.0

### PR DESCRIPTION
**Pull request by siccovansas**
This is my first time adding a new device so I might be doing some things wrong :). I tested this code with the [Idinio Zigbee LED Foot Dimmer](https://www.shada.nl/products/5450/zigbee-led-foot-dimmer) and it works (on, off, increase brightness, decrease brightness). When I control the foot dimmer directly by stepping on it the change its state is not received by Zigbee2MQTT though (there are also no log messages).

### CODE REVIEW

**Comment by avee87** [comment]
>  When I control the foot dimmer directly by stepping on it the change in state is not received by Zigbee2MQTT though (there are also no log messages).

This is likely because you need to add `configure` section and set up reporting. You can probably copypaste it from the device config I linked

File: devices/idinio.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const extend = require('../lib/extend');
+
+module.exports = [
+    {
+        zigbeeModel: ['Dimmer-Switch-ZB3.0'],
```
[the remark this answers is not in the document]
Comment (siccovansas) [reply devices/idinio.js]: Thanks! I removed `zigbeeModel` and added the `fingerprint`: https://github.com/siccovansas/zigbee-herdsman-converters/blob/900f59299571777a9924e86b93ee4541e71b3619/devices/idinio.js#L5

**Comment by siccovansas** [comment]
Thanks! Receiving/processing messages from the dimmer with the new code works great on the dev branch. I added a commit with the changes.

Everything seems to be working now :)!

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**